src/metabase/config.clj

```diff
@@ -99,7 +99,9 @@
      mb-version-info -> {:tag: \"v0.11.1\", :hash: \"afdf863\", :branch: \"about_metabase\", :date: \"2015-10-05\"}"
   (or (version-info-from-properties-file)
       ;; if version info is not defined for whatever reason
```

Comment:
We need some sort of def defaults here, otherwise snowplow will break. IDK if we have any sort of preferred values, but I think these defaults seem reasonable.